Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -764,6 +843,118 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
 
     #endregion
 
+    #region Quaternion Utils
+
+    /// <summary>
+    /// Converts the quaternion into an axis and an angle and returns the vector
+    /// axis * angle.
```

Comment:
Probably worth mentioning in the comment that the magnitude of the vector is measured in _degrees_ and not radians.  So the user better not try using this directly with PhysX angular velocity without a conversion for example.

Reply:
argh. I couldn't find any documentation for those units so I honestly wasn't sure how PhysX was measured. I should probably switch this to return radians for compatibility's sake.

Reply:
Actually, for the sake of not double-converting I'm going to leave it in degrees and just clarify the comment.